### PR TITLE
SOC: Add Access Control, Data Management and Secure Development policies

### PR DESCRIPTION
**Pull request by knolleary**
Part of https://github.com/flowforge/admin/issues/165

# Description

Adds initial drafts of the following security policies:

 - Access Control
 - Data Management
 - Secure Development

This PR targets the `soc-info-sec-aup` branch which introduces the initial policies - which this PR updates with relevant links.

**Please review the content, but do not merge yet**

### CODE REVIEW

File: src/handbook/company/security/access-control.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+keys, must meet the length, complexity, age, and rotation requirements defined
+in security standards.
+
+4. Use strong password and multi-factor authentication (MFA) whenever possible
+to authenticate to all computing resources.
+
+5. MFA is required to access any critical system or resource, including but not
+limited to resources in production environments.
+
+6. Unused accounts, passwords, access keys must be removed within 30 days
```
Comment (ZJvandeWeg) [inline src/handbook/company/security/access-control.md:43]:
```suggestion
6. Unused accounts, passwords, access keys must be removed within 30 days.
```

File: src/handbook/company/security/access-control.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+## Scope 
+
+This policy applies to all FlowForge information systems that process, store, or
+transmit confidential data as defined in the FlowForge [Data Management Policy](./data-management.md).
+
+It applies to all employees of FlowForge and to all external parties with access
+to FlowForge system resources.
+
+## Access Control Policy
```
Comment (ZJvandeWeg) [inline src/handbook/company/security/access-control.md:24]: Should also be part of the requirements for vendor selection?

File: src/handbook/company/security/access-control.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+
+### Access Authorization and Termination
+
+1. Access authorization shall be implemented using role-based access control
+(RBAC) or similar mechanism.
+
+2. Standard access based on a user's job role may be pre-provisioned during
+employee onboarding. All subsequent access requests to computing resources must
+be approved by the requestor’s manager, prior to granting and provisioning of
```
Comment (ZJvandeWeg) [inline src/handbook/company/security/access-control.md:58]: Link to how we get additional access? The request policy in the handbook?

File: src/handbook/company/security/access-control.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+access.
+
+3. Access to critical resources, such as production environments, must be
+approved by the CTO in addition to the requestor’s manager.
+
+4. Access must be reviewed on a regular basis and revoked if no longer needed.
+
+5. Upon termination of employment, all system access must be revoked and user
```
Comment (ZJvandeWeg) [inline src/handbook/company/security/access-control.md:66]: Should we start an off boarding template? So when we need it, this policy is actually effective? cc @Tracy-Anthony

File: src/handbook/company/security/access-control.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+6. All system access must be reviewed at least annually and whenever a user's
+job role changes.
+
+### Shared Secrets Management
+
+1. Use of shared credentials/secrets must be minimized and approved on an
+exception basis.
+
+2. If required by business operations, secrets/credentials must be shared
+securely and stored in the company provided password manager.
```
Comment (ZJvandeWeg) [inline src/handbook/company/security/access-control.md:78]:
```suggestion
securely and stored in a 1Password vault.
```

Comment (ZJvandeWeg) [reply src/handbook/company/security/access-control.md:78]: I'd like to be explicit on how to, to guide people to the best practises.

File: src/handbook/company/security/secure-development.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+All FlowForge applications and information systems that are business critical
+and/or process, store, or transmit Confidential data. This policy applies to all
+internal and external engineers and developers of FlowForge software and
+infrastructure.
+
+## Secure Development Policy
+
+FlowForge policy requires that:
+
+ 1. FlowForge software engineering and product development is required to follow
```
Comment (ZJvandeWeg) [inline src/handbook/company/security/secure-development.md:27]: cc @MarianRaphael Maybe you'll need to cross link this?

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

File: src/handbook/company/security/access-control.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+It applies to all employees of FlowForge and to all external parties with access
+to FlowForge system resources.
+
+## Access Control Policy
+
+1. Access to all computing resources, including servers, end-user computing
+  devices, network equipment, services and applications, must be protected by
+  strong authentication, authorization, and auditing.
+
+2. Interactive user access must be associated to an account or login unique to
```
[the remark this answers is not in the document]
Comment (knolleary) [reply src/handbook/company/security/access-control.md:30]: Once we have agreed the policy details, we'll raise the necessary issues to get into compliance.

File: src/handbook/company/security/access-control.md (hunk 4 above)
Comment (knolleary) [reply src/handbook/company/security/access-control.md:66]: We will eventually have a `Human Resource Security Policy` that I believe will cover this and we can add cross-links once it's in place.

Comment (Tracy-Anthony) [reply src/handbook/company/security/access-control.md:66]: Yes, we should. I started working on the initial bit of the policy already.

File: src/handbook/company/security/access-control.md (hunk 2 above)
Comment (knolleary) [reply src/handbook/company/security/access-control.md:24]: Vanta provide a separate `Third-Party Management Policy` that I think address this. Once we get there, we can cross-link.

File: src/handbook/company/security/access-control.md (hunk 3 above)
Comment (knolleary) [reply src/handbook/company/security/access-control.md:58]: I think it is going to depend on the system... 'can I get a zoom license' is a different request to 'can I get access to the production AWS account'. I want to be careful not to overload this.

File: src/handbook/company/security/access-control.md (hunk 5 above)
Comment (knolleary) [reply src/handbook/company/security/access-control.md:78]: Will do, with a slightly different wording. Going to keep the 'company provided' part as we don't want these in user's personal vaults.

**Comment by ZJvandeWeg** [comment]
@knolleary up to you to merge

**Pull request merged**